packages/frontend-v2/package.json

```diff
@@ -9,7 +9,11 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@chakra-ui/react": "^1.8.8",
+    "@emotion/react": "^11",
```

Comment:
and then also, im guessing we should use chakra props over emotion to style chakra components? so making that clear in the docs as well. 

when introducing multiple ways to accomplish the same task into a codebase, there need to be much more stringent guidelines for how to go about accomplishing things, so new devs know what the ground rules are. in general, it adds things to learn/read/be familiar with before being able to comfortable contribute to a codebase

Reply:
definitely!! my point with Chakra was actually to address using multiple ways of styling things in the first place. If you look at the brief Chakra section in the document, it's like the first thing I mention :) 

But yeap I agree that documenting it and making it clear is most important!

Reply:
awesome, thank you!